src/bulk-load.js

```diff
@@ -125,6 +164,17 @@ module.exports = class BulkLoad extends EventEmitter {
       sql += '[' + c.name + '] ' + (c.type.declaration(c));
     }
     sql += ')';
+
+    // filter bulkOptions that are true and get their corresponding T-SQL keyword
```

Comment:
What do you think about removing the need to use `filter` and `map` by being a bit more explicit here?

```js
const addOptions = [];

if (this.bulkOptions.checkConstraints) {
  addOptions.push('CHECK_CONSTRAINTS');
}

if (this.bulkOptions.fireTriggers) {
  addOptions.push('FIRE_TRIGGERS');
}

if (this.bulkOptions.keepNulls) {
  addOptions.push('KEEP_NULLS');
}

if (this.bulkOptions.lockTable) {
  addOptions.push('TABLOCK');
}
```

Reply:
This would also allow you to remove the `bulkOptionMapping` property.

Reply:
Since its just 4 options, being explicit should be fine 👍